superset/viz_sip38.py

```diff
@@ -436,6 +436,12 @@ def cache_key(self, query_obj, **extra):
         cache_dict["extra_cache_keys"] = self.datasource.get_extra_cache_keys(query_obj)
         cache_dict["rls"] = security_manager.get_rls_ids(self.datasource)
         cache_dict["changed_on"] = self.datasource.changed_on
+
+        url_params = self.form_data.get("url_params")
+
+        if url_params:
+            cache_dict["url_params"] = url_params
+
```

Comment:
same here (thanks for keeping this in sync)

Reply:
See ^^.